### PR TITLE
Fixed issue with duplicate method

### PR DESCRIPTION
**Pull request by TijanAyo**

### CODE REVIEW

File: src/endpoints/endpoints.service.ts [hunk 1]
```diff
@@ -28,14 +28,17 @@ export class EndpointsService {
   ): Promise<Endpoint> {
     try {
       const endpoint = await this.endpointRepository.findOne({
-        where: { name: createEndpointDto.name },
+        where: {
+          method: createEndpointDto.method,
+          route: createEndpointDto.route
+        }
```
Comment (Chimexx) [inline src/endpoints/endpoints.service.ts:34]: I think there is a possibility of two or more endpoints having the exact same method and route name (when they belong to different apis) . So you should narrow it down further using the apiId

**Review by Chimexx — CHANGES_REQUESTED** [review]

File: src/endpoints/endpoints.service.ts (hunk 1 above)
Comment (TijanAyo) [reply src/endpoints/endpoints.service.ts:34]: That is what I tried narrowing down... once the user creates a route that has the same method... it should prompt out an error

File: src/endpoints/endpoints.service.ts [hunk 2]
```diff
@@ -28,14 +28,18 @@ export class EndpointsService {
...
       });
 
       if (endpoint) {
         throw new BadRequestException(
           ZapiResponse.BadRequest(
             'Existing Endpoint',
-            'An endpoint with this name already exists, use another name',
+            'An endpoint with duplicate method already exists, use another method',
```
Comment (Chimexx) [inline src/endpoints/endpoints.service.ts:42]: Will be nice if you leave this as 'This endpoint already exists'

**Review by Chimexx — APPROVED** [review]

File: src/endpoints/endpoints.service.ts (hunk 2 above)
Comment (TijanAyo) [reply src/endpoints/endpoints.service.ts:42]: I will do that...

**Pull request merged**